base/src/org/compiere/model/MBankStatement.java

```diff
@@ -712,4 +712,37 @@ public boolean isComplete()
 			|| DOCSTATUS_Reversed.equals(ds);
 	}	//	isComplete
 	
+	/**
+	 * 	Get BankStatement (CashJournal) for Bank Account and date
+	 *	@param ctx context
+	 *	@param C_BankAccount_ID cashbook
+	 *	@param dateAcct date
+	 *	@param trxName transaction
+	 *	@return The Bank Statement (Cash Journal)
+	 */
+	public static MBankStatement get (Properties ctx, int C_BankAccount_ID, 

```

Comment:
What is the reason for add this method?
If it is for create a bank statement automatically then already exists a static method for handle it:
<pre>
static public MBankStatementLine addPayment(MPayment payment)
</pre>

Reply:
Your right.  I didn't see that.  I'll make the correction.

Reply:
Looking closer, I did use the static method in MPayment - new line 1887.  I think this method should have been deleted.  Again, I'll make the change and update the pull request.